base/applications/rapps/include/rosui.h

```diff
@@ -116,46 +120,37 @@ class CPointerArray
         return DPA_Sort(m_hDpa, s_OnCompareItems, (LPARAM)this);
     }
 
-    INT Search(T* item, INT iStart, UINT uFlags)
+    INT Search(T* item, INT iStart = 0, UINT uFlags = 0)
     {
-        return DPA_Search(m_hDpa, item, 0, s_OnCompareItems, (LPARAM)this, 0);
+        return DPA_Search(m_hDpa, item, iStart, s_OnCompareItems, (LPARAM)this, uFlags);
     }
 };
 
-class CUiRect
```

Comment:
OK this is a class that doesn't serve any purpose, since it just duplicates the `RECT` structure (and that structure is manipulated anyways anywhere else in the code).
Since this rosui.h header is also used exclusively in ATL apps it could, at best, use the `CRect` structure from `atltypes.h`, in case of need.

Reply:
It does not duplicate the RECT structure, it provides c++ initialization for the RECT structure.

Reply:
Then it should be possible to just use the CRect from ATL? (but again, this doesn't look like it's necessary to be used as other code just uses the RECT without any problems)